.gitignore

```diff
@@ -1,16 +1,8 @@
-*.pyc
-*.pyo
-*~
+*.egg-info
+*.py[co]
 *.sw[po]
+*~
 .DS_Store
-.coverage
-.installed.cfg
-MANIFEST
```

Comment:
Not specifically, I can't remember. it's just I had to add this on some projects in the past, maybe it's not the case anymore.

Forget about my remark, then ;)